opsdroid/core.py

```diff
@@ -207,9 +208,22 @@ def setup_skills(self, skills):
         """
         for skill in skills:
             for _, func in skill["module"].__dict__.items():
+                if isinstance(func, type) and issubclass(func, Skill) and func != Skill:
+                    skill_obj = func(self, skill['config'])
+
+                    for name in skill_obj.__dir__():
+                        try:
+                            method = getattr(skill_obj, name)
+                        except Exception:
```

Comment:
Can you make this less broad?

Reply:
Actually, no. Imagine this code:

```python
class MySkill(Skill):
    @property
    def erroneous_property(self):
        return self.sdk_this_skill_uses.get_erroneous_property()
```

Where `get_erroneous_property()` can raise any type of exception. Been there, done that :disappointed:

We can either remove the try/except altogether and ask skill writers to look out for this corner case, or leave the except block this broad.

Reply:
Fair enough! Can you please add a pylint disable with this justification in a commend. See [here](https://github.com/opsdroid/opsdroid/blob/7df9dc03b47bbb5b124ad6b7f74d02671058c60f/opsdroid/core.py#L272) for an example of this.